packages/eslint-plugin-query/docs/rules/prefer-query-object-syntax.md

```diff
@@ -0,0 +1,53 @@
+# Prefer object syntax for useQuery (`@tanstack/query/prefer-query-object-syntax`)
+
+You can use [`useQuery`](https://tanstack.com/query/v4/docs/reference/useQuery) in two different ways.
```

Comment:
When v5 is out, we should update the link here

Reply:
when v5 is out, we can probably delete this rule because there will only be the query syntax 🤔. Or, we'll keep it so that people can use the auto fix to migrate and then delete it later...